ci-operator/step-registry/optional-operators/ci/azure/ovn/hybrid/optional-operators-ci-azure-ovn-hybrid-workflow.yaml

```diff
@@ -0,0 +1,15 @@
+workflow:
+  as: optional-operators-ci-azure-ovn-hybrid
+  steps:
+    pre:
+    - chain: ipi-conf-azure
+    - ref: ovn-conf
+    - ref: ovn-conf-hybrid-manifest
```

Comment:
I'm wondering if we should create a hybrid-ovn pre and post job for each cloud provider to reduce code duplication. The existing hybrid ovn workflow would then call those pre and posts. The regular IPI jobs do that. What do you think?

Reply:
sure that would make this cleaner, but we would still need a pre and post for each cloud provider with hybrid ovn and the workflows without hybrid-ovn for each cloud provider are using the pre and post ipi chains anyway, so I am not sure that will remove code duplication.